File: 프로그래밍/c/code/03.논리.c
// ---
// check: none
// ---
int score = 85;
printf("%d\n", score >= 80 && score < 90);   // 1   둘 다 참이어야 참
printf("%d\n", score < 60 || score > 90);    // 0   하나만 참이어도 참
printf("%d\n", !(score >= 80));              // 0   뒤집는다

File: 프로그래밍/py/code/03.논리.py
# ---
# check: none
# ---
score = 85
print(score >= 80 and score < 90)   # True    둘 다 참이어야 참
print(score < 60 or score > 90)     # False   둘 다 거짓이라 거짓 (하나라도 참이면 참)
print(not score >= 80)              # False   뒤집는다
